celements-model/src/main/java/com/celements/model/util/DefaultModelUtils.java

```diff
@@ -217,6 +219,18 @@ public String serializeRefLocal(EntityReference ref) {
     return serializeRef(ref, ReferenceSerializationMode.LOCAL);
   }
 
+  @Override
+  public String normalizeLang(String lang) {
+    String ret = nullToEmpty(lang).trim();
+    if (!"default".equals(ret)) {
+      lang = Util.normalizeLanguage(ret);
+      if ("default".equals(ret)) {
```

Comment:
`ret` will not be changed since line 225 thus this if is always false. This does not make sense to me.